docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -278,6 +278,52 @@ When using theme.json in a block or classic theme, these settings will stop the
 
 To enable something from the above, just set whatever value you want to change to `true` for more granularity.
 
+## Limiting interface options with theme.json filters
+
+The theme.json file is a great way to control interface options, but it only allows for global or block-level modifications, which can be limiting in some scenarios.
+
+For instance, in the previous section, color and typography controls were disabled globally using theme.json. But let's say you want to enable color settings for users who are Administrators. 
+
+To provide more flexibility, WordPress 6.1 introduced server-side filters allowing you to customize theme.json data at four different data layers.
+
+- `wp_theme_json_data_default` - Hooks into the default data provided by WordPress
+- `wp_theme_json_data_blocks` - Hooks into the data provided by blocks.
+- `wp_theme_json_data_theme` - Hooks into the data provided by the current theme.
+- `wp_theme_json_data_user` - Hooks into the data provided by the user.
+
+In the following example, the data from the current theme's theme.json file is updated using the `wp_theme_json_data_theme` filter. Color controls are restored if the current user is an Administrator.
+
+```
+// Disable color controls for all users except Administrators.
+function example_filter_theme_json_data_theme( $theme_json ){
+    $is_admin = current_user_can( 'edit_theme_options' );
+
+    if ( $is_admin ) {
```

Comment:
Small nit: At first, as I glanced through this quickly, I thought you meant the function is_admin().  Can we use a more unique variable name?

Reply:
Agreed :) Also stumbled over this. Appart from that it is a great addition :) 

Maybe even renaming it to `is_administrator` helps clarify it a little more.

Reply:
Good call. Thanks for reviewing!